index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,500&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <nav><img src="assets/ic_menu.png" alt="menu icon"></nav>
```

Comment:
- [ ] [OPTIONAL]: Please consider putting `hamburger` menu inside a `<button>` tag
- [ ] [OPTIONAL]: Please check the `hamburger` menu, it changed it's location and immigrated to the left side far away 😄 

![image](https://user-images.githubusercontent.com/77038610/143488504-a9a281d8-248e-461a-9976-18bdc7264d2c.png)

Reply:
thanks for the suggestions. I appreciate it